protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -499,6 +500,45 @@ service TradingDataService {
     // List all entities created by transaction hash
     rpc ListEntities(ListEntitiesRequest) returns (ListEntitiesResponse);
 
+    // Export Network History
+    //
+    // Extract CSV table data from network history between two block heights.
+    //
+    // The requested block heights must fall on network history segment boundaries which can
+    // be discovered calling the API to list all network history segments. By default
+    // segments contain 1000 blocks, if so ranges such as (1, 1000), (1001, 2000), (1, 3000)
+    // would all fall on segment boundaries and be valid.
+    //
+    // In gRPC, results are returned in a chunked stream of base64 encoded CSV data
+    //
+    // Through the REST gateway, the base64 data chunks are decoded and streamed as a
+    // `content-type: text/csv` HTTP response.
+    //
+    // CSV data is escaped & quoted in the as described in the postgresql documentation at 
```

Comment:
well sort-of; CSV isn't really a standard - there are quite a few different conventions for specifying how to escape and quote special characters, newlines, NULL values etc.. 

We're using the postgres standard because, well, that's how it comes to us and its a reasonable one. I could copy-paste what's in the postgres docs instead of linking to it if you prefer?

Reply:
I think the origin of the standard doesn't matter as much as what the standard is - so copying out what it is seems more helpful to me :)